app/api/v1/admin/patients.rb

```diff
@@ -0,0 +1,109 @@
+module V1
+  module Admin
+    class Patients < API
+      patients_crud_url = '/api/v1/admin/patients'
+      resources :patients do
+        desc 'Return all patients'
+        get do
+          patients = Patient.all
+          present patients, with: Entities::Patient
+        end
+
+        desc 'Return a specific patient'
+        route_param :id do
+          get do
+            patient = Patient.find(params[:id])
+            present patient, with: Entities::Patient
+          rescue ActiveRecord::RecordNotFound
```

Comment:
let's get rid of errors' handling since we agreed to move to a general handler 

check [the ticket](https://trello.com/c/wGRbj1hx/70-add-error-handling-for-crud-operations) for these purposes pls

Reply:
I agree in this case. Does this apply to all error handling? Because in my CRUD there are several special cases of handling typical errors (for example, when searching for allergies).